packages/components-organizations-registrar/src/pages/services/utils/index.js

```diff
@@ -0,0 +1,49 @@
+export const getTitle = (step) => {
```

Comment:
i'd prefer separate files for each util. so that they are easily loaded

Reply:
@sloops77 Fixed